packages/ts/react-form/src/index.ts

```diff
@@ -17,18 +18,21 @@ import {
 } from '@hilla/form';
 import type { BinderNode } from '@hilla/form/BinderNode.js';
 import { useEffect, useMemo, useReducer, useRef } from 'react';
+import type { VaadinWindow } from './types.js';
 
-const $wnd = window as any;
+declare const __VERSION__: string;
+
+const $wnd = window as VaadinWindow;
 
 $wnd.Vaadin ??= {};
 $wnd.Vaadin.registrations ??= [];
 $wnd.Vaadin.registrations.push({
   is: '@hilla/react-form',
-  version: /* updated-by-script */ '2.2.0',
+  version: /* updated-by-script */ '2.3.0-alpha6',
```

Comment:
```suggestion
  version: __VERSION__,
```

If there is any other obsolete version specification, we should address it as well.